ios/Video/RCTVideo.swift

```diff
@@ -307,6 +307,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     // MARK: - Progress
 
     func sendProgressUpdate() {
+        guard onVideoProgress != nil else { return }
+
```

Comment:
I think you will introduce a regression linked to ads,
please see next lines: 

```swift
            #if USE_GOOGLE_IMA
                if !_didRequestAds && currentTimeSecs >= 0.0001 && _adTagUrl != nil {
                    _imaAdsManager.requestAds()
                    _didRequestAds = true
                }
            #endif
```

Reply:
Good catch! I have made it to skip "onVideoProgress != nil" when using ads as we need this code to run in this situation
```swift
#if !USE_GOOGLE_IMA
  guard onVideoProgress != nil else { return }
#endif
```